myfitnesspal/entry.py

```diff
@@ -35,3 +50,17 @@ def __unicode__(self):
             self.name,
             self.nutrition_information,
         )
+
+    @property
+    def short_name(self):
+        if self._short_name:
+            return self._short_name.strip()
```

Comment:
Good catch!  I hadn't considered that my suggested changes might cause this to be a problem.